<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>截图器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="main">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>截图</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>延迟时间(S)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="delay_value">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>过渡补偿(S)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="transit_value">
            <property name="singleStep">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>文件格式</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="file_type"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="isTop">
            <property name="text">
             <string>保持置顶窗口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,10">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>项目</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="subject"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="shot">
            <property name="text">
             <string>截图</string>
            </property>
            <property name="icon">
             <iconset resource="pic.qrc">
              <normaloff>:/pic/shot.png</normaloff>:/pic/shot.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
       <zorder></zorder>
       <zorder></zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="subject_list">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sunject_name"/>
          </item>
          <item>
           <widget class="QPushButton" name="add_item">
            <property name="text">
             <string>添加项目</string>
            </property>
            <property name="icon">
             <iconset resource="pic.qrc">
              <normaloff>:/pic/add.png</normaloff>:/pic/add.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>333</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open_dir"/>
    <addaction name="subject_dir"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <action name="open_dir">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/open_dir.png</normaloff>:/pic/open_dir.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/exit.png</normaloff>:/pic/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="subject_dir">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/subject_dir.png</normaloff>:/pic/subject_dir.png</iconset>
   </property>
   <property name="text">
    <string>打开项目文件夹
</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
